src/arkode/arkode_lsrkstep_io.c

```diff
@@ -0,0 +1,661 @@
+  Does not change problem-defining function pointers or
+  user_data pointer.
+  ---------------------------------------------------------------*/
+int lsrkStep_SetDefaults(ARKodeMem ark_mem)
+{
+  ARKodeLSRKStepMem step_mem;
+  int retval;
+  long int lenrw, leniw;
+
+  /* access ARKodeLSRKStepMem structure */
+  retval = lsrkStep_AccessStepMem(ark_mem, __func__, &step_mem);
+  if (retval != ARK_SUCCESS) { return (retval); }
+
+  /* Remove current SUNAdaptController object, and replace with "PI" */
+  if (ark_mem->hadapt_mem->owncontroller)
+  {
+    retval = SUNAdaptController_Space(ark_mem->hadapt_mem->hcontroller, &lenrw,
+                                      &leniw);
+    if (retval == SUN_SUCCESS)
+    {
+      ark_mem->liw -= leniw;
+      ark_mem->lrw -= lenrw;
+    }
+    retval = SUNAdaptController_Destroy(ark_mem->hadapt_mem->hcontroller);
+    ark_mem->hadapt_mem->owncontroller = SUNFALSE;
+    if (retval != SUN_SUCCESS)
+    {
+      arkProcessError(ark_mem, ARK_MEM_FAIL, __LINE__, __func__, __FILE__,
+                      "SUNAdaptController_Destroy failure");
+      return (ARK_MEM_FAIL);
+    }
+  }
+  ark_mem->hadapt_mem->hcontroller = SUNAdaptController_PI(ark_mem->sunctx);
+  if (ark_mem->hadapt_mem->hcontroller == NULL)
+  {
+    arkProcessError(ark_mem, ARK_MEM_FAIL, __LINE__, __func__, __FILE__,
+                    "SUNAdaptControllerPI allocation failure");
+    return (ARK_MEM_FAIL);
+  }
+  ark_mem->hadapt_mem->owncontroller = SUNTRUE;
+  retval = SUNAdaptController_Space(ark_mem->hadapt_mem->hcontroller, &lenrw,
+                                    &leniw);
+  if (retval == SUN_SUCCESS)
+  {
+    ark_mem->liw += leniw;
+    ark_mem->lrw += lenrw;
+  }
+
+  /* Set default values for integrator optional inputs
+     (overwrite some adaptivity params for LSRKStep use) */
+  step_mem->req_stages = 0; /* no stages */
+
+  /* Spectral info */
+  step_mem->lambdaR      = 0;
+  step_mem->lambdaI      = 0;
+  step_mem->sprad        = 0;
+  step_mem->spr_max      = 0;
+  step_mem->spr_min      = 0;
+  step_mem->dom_eig_sfty = 1.01;
+  step_mem->dom_eig_freq = 25;
+
+  /* Flags */
+  step_mem->new_dom_eig        = SUNTRUE;
+  step_mem->const_Jac          = SUNFALSE;
+  step_mem->dom_eig_is_current = SUNFALSE;
+  step_mem->is_SSP             = SUNFALSE;
+
+  ark_mem->hadapt_mem->adjust = 0;               /* set default adjustment */
+  ark_mem->hadapt_mem->etamxf = SUN_RCONST(0.3); /* max change on error-failed step */
+  ark_mem->hadapt_mem->safety = SUN_RCONST(0.99); /* step adaptivity safety factor  */
+  ark_mem->hadapt_mem->growth = SUN_RCONST(25.0); /* step adaptivity growth factor */
+
+  (void)SUNAdaptController_SetErrorBias(ark_mem->hadapt_mem->hcontroller,
+                                        SUN_RCONST(1.2));
+  (void)SUNAdaptController_SetParams_PI(ark_mem->hadapt_mem->hcontroller,
+                                        SUN_RCONST(0.8), -SUN_RCONST(0.31));
```

Comment:
Are these different from the standard defaults? If so, this should be noted in the documentation.

Reply:
Given that the "standard defaults" are currently in flux, then this is clear non-default.  I agree that the LSRKStep documentation should mention that it defaults to the PI controller with these exponents, error bias, safety, growth, and step failure factors.

Reply:
The default values of these parameters are the same as those in `ERKStep` and are not explicitly mentioned there either. Where do you think would be the appropriate place to document their default values?

Reply:
I think in the section on [optional inputs for time step adaptivity](https://sundials.readthedocs.io/en/latest/arkode/Usage/User_callable.html#optional-inputs-for-time-step-adaptivity) would be the right place to document the defaults.

Reply:
The ERKStep documentation mentions that it defaults to the PI controller [here](https://sundials.readthedocs.io/en/latest/arkode/Usage/ERKStep/User_callable.html#c.ERKStepSetAdaptController); however, that function is marked as deprecated, and so the information should be added elsewhere to specify that ERKStep selects a different controller than the default for ARKODE.

I like @gardner48's suggestion of where to put this discussion.  Specifically, I recommend that we expand the "Note" for `ARKodeSetAdaptController` to discuss the ARKODE default, as well as any stepper-specific defaults. 

I'll also point out that in the near future, @Steven-Roberts , @samihere2018, and I will be updating the default controllers across the board for ARKODE, so it will be helpful if most of the relevant discussion  is in this one location.

Reply:
The default controller for LSRKStep is removed. It inherits the controller from ARKode now. It seems that ARKode controller performs slightly better. 

I will resolve this conversation since it is irrelevant now until the new PR for the new default controllers.